app/views/devise/sessions/new.html.haml

```diff
@@ -1,15 +1,19 @@
+- flash.each do |name, msg|
+  = content_tag :div, msg, :id => "flash_#{name}" if msg.is_a?(String)
```

Comment:
@vakyl9r @Magons IMPORTANT! There is no error message on Sign In page without this code!